src/python/WMCore/WorkQueue/Policy/Start/Block.py

```diff
@@ -45,15 +45,22 @@ def split(self):
                         parentList[dbsBlock["Name"]] = self.sites
                     else:
                         parentList[dbsBlock["Name"]] = self.siteDB.PNNstoPSNs(dbsBlock['PhEDExNodeList'])
-
+            
+            if self.args['SliceType'] == 'NumberOfEvents' and block['NumberOfEvents'] == 0:
+                # if there is 0 events file and use EventAwareLumiBased algo
+                # set the estimate jobs to 1 (It is hard to get the realistic estimate 
+                # unless run the splitting alogrithm
+                estimateJobs = 1
+            else:
+                estimateJobs = ceil(float(block[self.args['SliceType']]) / float(self.args['SliceSize'])),
```

Comment:
Are these args already integers or are you converting from string? A quick search shows that you could get rid of all kinds of casting to floats simply by importing division from the future.